studentWork/Unit1/HelloWorld.java

```diff
@@ -0,0 +1,11 @@
+package studentWork.Unit1;
```

Comment:
Added by mistake? I think we would want to avoid putting any `studentWork/...` files in the upstream.

Reply:
Yikes, did not mean to add this. Will try to fix - thanks for taking a look and catching this!